### PR TITLE
Works about

### PR DESCRIPTION
**Pull request by ayebaishmo**
# In this project:
- Create two sections works and about
- Design work section with css grid and flex box
- Design about section with flexbox and ul li tags for list items
- Use background images with no repeat property

# Attarched is screenshoot of no lint errors beacuse there seems to be an error with Githuub Action for webhint. Thankyou 
No_linter_errors

### CODE REVIEW

**Review by fernandorpm — CHANGES_REQUESTED** [review]
# :x: Pull Request Invalid! :x:
Hello there!

I can see that you have been working throughout the project, but you need to complete the following prerequisites in order to start the Reviewing process!



## Invalid PR - Required before Reviewing
> These prerequisites HAVE TO be done in order for your project to be reviewed. This means that the project doesn't attend to Microverse's Standard yet. I know that it can be frustrating, but they should be seen as part of your path to become an even better Developer!

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready

## Wrong linter usage

- Please be sure to add the [required linters](https://github.com/microverseinc/linters-config) to this project.

- If you compare your `.hintrc` file with [the one in Microverse's repo](https://github.com/microverseinc/linters-config/blob/master/html-css/.hintrc), you will notice that they are very different. Please update your code with the correct one. Once you do it, your linters should pass.

> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Just keep in mind that Microverse has a standard for the project requirements, to guarantee that you will be able to become a great developer, and both you and I have to follow them!

___
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

Cheers and Happy Coding! :heart:

> *As described in the [Code reviews limit Policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of code reviews per project (you can check the exact number in your Dashboard). If you think that the code review was not fair, you can [request a second opinion using this form](https://airtable.com/shrQAqnBwek5a0O0s).*

**Comment by ayebaishmo** [comment]
> 
> # ❌ Pull Request Invalid! ❌
> 
> Hello there!
> 
> I can see that you have been working throughout the project, but you need to complete the following prerequisites in order to start the Reviewing process!
> 
> 
> ## Invalid PR - Required before Reviewing
> 
> > These prerequisites HAVE TO be done in order for your project to be reviewed. This means that the project doesn't attend to Microverse's Standard yet. I know that it can be frustrating, but they should be seen as part of your path to become an even better Developer!
> 
> Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready
> ## Wrong linter usage
> 
>     * Please be sure to add the [required linters](https://github.com/microverseinc/linters-config) to this project.
> 
>     * If you compare your `.hintrc` file with [the one in Microverse's repo](https://github.com/microverseinc/linters-config/blob/master/html-css/.hintrc), you will notice that they are very different. Please update your code with the correct one. Once you do it, your linters should pass.
> 
> 
> > Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Just keep in mind that Microverse has a standard for the project requirements, to guarantee that you will be able to become a great developer, and both you and I have to follow them!
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> Cheers and Happy Coding! ❤️
> 
> > _As described in the [Code reviews limit Policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of code reviews per project (you can check the exact number in your Dashboard). If you think that the code review was not fair, you can [request a second opinion using this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Hello,  
I have checked my .hintrc file and actually recoped the same code but still it brings that error.
Secondly, We are couple of people facing that same lint error, cause now am stack cannot get passed  it. Any more information about it can be of help.
Thankyou

File: index.html [hunk 1]
```diff
@@ -36,5 +36,147 @@ <h2>I’am a Software Developer</h2>
...
+                <h4>Multi-Post Stories</h4>
+                <p>A daily selection of privately personalized reads; 
+                    no accounts or sign-ups required. has been the industry's standard
+                dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+                <ul>
+                    <li><button type="button">css</button></li>
+                    <li><button type="button">html</button></li>
+                    <li><button type="button">bootstrap</button></li>
+                    <li><button type="button">Ruby</button></li>
+                </ul>
```
Comment (bimbolabuari) [inline index.html:58]: As stated in the project requirement, Kindly implement the button interactions (enable, hover, etc.) 👍

**Review by bimbolabuari — CHANGES_REQUESTED** [review]
Hi @ayebaishmo 🙂 , 

You have done a good job so far 🚀  

## Highlights 🎉 

- Nice UI design 🎉 
- Good Github workflow 🎉 
- Descriptive readme 🎉 

There are some issues that you still need to work on to go to the next project but you are almost there! 🎉 

## Required Changes ♻️
_Kindly check the comments under the review._

## Optional suggestions
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

--------


_Reviewed by  @bimbolabuari_

**Review by mke2111 — APPROVED** [review]
### Hi @ayebaishmo, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
**Congratulations! 🎉** 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**